pyproject.toml

```diff
@@ -202,6 +202,7 @@ disable = [
     "useless-return",                    # PLR1711
     # "no-self-use", # PLR6301  # Optional plugin, not enabled
 ]
+max-positional-arguments = 10
```

Comment:
this needs to go under 'tool.pylint.DESIGN' like discussed in the matter server conversation

Reply:
It does work here too (as you can see from the tests), but agreed, according to https://pylint.pycqa.org/en/latest/user_guide/configuration/all-options.html#max-positional-arguments it is under design. Seems to be "more" correct then? :thinking: :shrug: